src/dbos-runtime/cli.ts

```diff
@@ -48,7 +50,8 @@ program
   .option('-p, --port <number>', 'Specify the port number')
   .option('-l, --loglevel <string>', 'Specify log level')
   .option('-c, --configfile <string>', 'Specify the config file path', dbosConfigFilePath)
-  .option('-e, --entrypoint <string>', 'Specify the entrypoint file path')
+  .option('-e, --entrypoint <string>', 'Specify the entrypoint file path [DEPRECATED]')
```

Comment:
I don't think should should be configurable from the command line. My suggestion would be to remove the `entrypoints` flag, put a deprecation message on `entrypoint`, and remove it down the line. Entrypoints should only be configurable from the configuration file.

Reply:
I just nuked it entirely